azurerm/internal/services/network/application_gateway_resource.go

```diff
@@ -821,6 +821,12 @@ func resourceArmApplicationGateway() *schema.Resource {
 							}, true),
 						},
 
+						"port": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							ValidateFunc: validate.PortNumber,
+						},
```

Comment:
_minor_ this is sorted Required fields then Optional fields (and then alphabetically) - could we move this below so it's consistent?

can we also [add this new field to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/application_gateway.html.markdown#L343-L355)?

Reply:
Fixed. Though it seems the Optional fields are not sorted alphabetically.